docs/source/authoring_flows.rst

```diff
@@ -519,17 +519,190 @@ Example Flows
+                    "endpoint_id.$": "$.destination.id"
+                },
+                "ResultPath": "$.DestinationPathInfo"
+            },
+            "SetDestinationInfo": {
+                "Next": "Transfer",
+                "Type": "ExpressionEval",
+                "Comment": "Set information about the destination path",
+                "Parameters": {
+                    "exists.=": "is_present('DestinationPathInfo.details.DATA[0]')",
+                    "is_folder.=": "getattr('DestinationPathInfo.details.DATA[0].is_folder', False)",
+                    "destination_file.=": "getattr('DestinationPathInfo.details.DATA[0].name', '/')",
+                    "destination_folder.=": "DestinationPathInfo.details.path"
+                },
+                "ResultPath": "$.DestinationInfo"
+            },
+            "Transfer": {
+                "Next": "Delete",
+                "Type": "Action",
+                "Comment": "Run the initial transfer operation from the source ep/source path to the destination ep/destination path",
+                "WaitTime": 172800,
+                "ActionUrl": "https://actions.globus.org/transfer/transfer",
+                "Parameters": {
+                    "label.=": "getattr('transfer_label', 'Transfer for Move Flow Run with id ' + _context.run_id)",
+                    "transfer_items": [
+                        {
+                            "recursive.$": "$.SourceInfo.is_recursive",
+                            "source_path.$": "$.source.path",
+                            "destination_path.=": "(destination.path + '/' + SourceInfo.source_file) if (DestinationInfo.is_folder or ((not DestinationInfo.exists) and SourceInfo.is_recursive)) else destination.path"
+                        }
+                    ],
+                    "source_endpoint_id.$": "$.source.id",
+                    "destination_endpoint_id.$": "$.destination.id"
+                },
+                "ResultPath": "$.TransferResult",
+            },
+            "Delete": {
+                "End": true,
+                "Type": "Action",
+                "Comment": "Use Transfer to delete the initial source ep/source path. It uses the same value for recursive as the transfer",
+                "WaitTime": 172800,
+                "ActionUrl": "https://actions.globus.org/transfer/delete",
+                "Parameters": {
+                    "items.=": "[source.path]",
+                    "label.=": "getattr('delete_label', 'Delete from Source for Move Flow Run with id ' + _context.run_id)",
+                    "recursive.$": "$.SourceInfo.is_recursive",
+                    "endpoint_id.$": "$.source.id"
+                },
+                "ResultPath": "$.DeleteResult",
+            }
+        },
+        "Comment": "A Flow for performing a logical 'move' operation by first transferring from a source to a destination and then deleting from the source",
+        "StartAt": "LookupSourcePath"
```

Comment:
These two should be at the top of the flow def for documentation sake.
If these are the reference examples people will use, the first thing they run into being a keyword telling them where to start in a big complex json blob is helpful